src/Sfa.Tl.ResultsAndCertification.Application/Mappers/CoreRommExtractMapper.cs

```diff
@@ -35,6 +35,8 @@ private string GetAssessmentSeriesYear(ICollection<TqPathwayAssessment> tqPathwa
 
         private string GetCurrentCoreGrade(TqRegistrationPathway tqRegistrationPathway)
         {
+            string coreGrade = string.Empty;
```

Comment:
Do we need string coreGrade=string.empty here ?

Reply:
Done